db/migrate/20230901073522_rename_user_id_to_author_id_in_likes.rb

```diff
@@ -0,0 +1,5 @@
+class RenameUserIdToAuthorIdInLikes < ActiveRecord::Migration[7.0]
+  def change
+    rename_column :likes, :user_id, :author_id
+  end
+end
```

Comment:
- [ ] this migration file is stopping your migration from completing, kindly look closer to it
![image](https://github.com/kayjeee/Kayjeee_Postgresql_blogapp/assets/66097371/d77485bd-f4ae-4291-a14c-7dfa347e5a66)